cfpb/cfgov-refresh
# Email signup

## Additions

- Enabled gov delivery subscriptions to work on wagtail pages

## Changes

- randomized formfieldWithButton Id and included name field for post requests.

## Testing
- Load refresh Db
- **Set gov delivery & subscription  environmental variables** (login into refresh for creds)
- Visit [here](http://localhost:8000/policy-compliance/rulemaking/regulatory-agenda/) and scroll to the button to subscribe your email.
![screen shot 2016-04-04 at 11 46 23 am](https://cloud.githubusercontent.com/assets/254877/14253776/e3939098-fa5a-11e5-84ec-9b270d68d5e8.png)
- Should see success screen
![screen shot 2016-04-04 at 11 45 38 am](https://cloud.githubusercontent.com/assets/254877/14253828/1b5d59a0-fa5b-11e5-8754-8369efc51d4e.png)



## Review

- @kurtw 
- @richaagarwal
